modules/custom/az_seo/az_seo.install

```diff
@@ -12,3 +12,21 @@ function az_seo_update_last_removed() {
   // Remove updates added before 2.6.0.
   return 9201;
 }
+
+/**
+ * Install az_news, az_person, az_event, az_flexible_page modules.
+ *
+ * Install modules that contain configuration dependencies.
+ *
+ * @see https://github.com/az-digital/az_quickstart/issues/3498
+ * @see https://github.com/az-digital/az_quickstart/pull/3499
+ */
+function az_seo_update_1021101() {
```

Comment:
Do we want this to be a 2.10.x update as well?  If so, we should probably update this number.
```suggestion
function az_seo_update_1021001() {
```